eisbuk-admin/src/hooks/useCountdown.ts

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect, useRef } from "react";
+import { DateTime } from "luxon";
+
+type CountdownTuple = [string, string, string, string];
+
+interface CountdownFunction {
+  (countdownDate: DateTime | null): CountdownTuple | null;
+}
+
+/**
+ * Counts down to provided `countdownDate`, ticks each second
+ * and returns a `CountdownTuple`
+ * @param {DateTime} countdownDate date we're counting down to
+ * @returns {string[]} tuple of four two-character `string` representations of
+ * countdown values in format: ["dd", "hh", "mm", "ss"]
+ */
+const useCountdown: CountdownFunction = (countdownDate) => {
+  const [countdown, setCountdown] = useState<CountdownTuple | null>(
+    getCountdownValues(countdownDate)
+  );
+
+  // store timeout in a ref to be accessible from multiple functions
+  const timeout = useRef<NodeJS.Timeout | null>(null);
+
+  // clear timeout and reset it to `null`
+  const resetTimeout = () => {
+    if (timeout.current) {
+      clearTimeout(timeout.current);
+      timeout.current = null;
+    }
+  };
+
+  // control 'tick' each second using setTimeout
+  useEffect(() => {
+    const hourInMillis = 1000;
```

Comment:
I think we should trigger this evary hour, instead of every second. I guess the variable name was updated but its value still needs to be changed to 3.6M.

Reply:
Right...I wanted to preserve the general functionality...will make it so that you can specify the tick length (`"hour"`, `"minute"`, `"second"`). I don't think it makes sense to leave it for day.